tools/stylelint-no-missing-var/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "stylelint-no-missing-var",
+  "version": "1.0.1",
+  "license": "MIT",
+  "author": "Rajdeep Chandra",
+  "main": "index.js",
+  "scripts": {
+    "lint": "stylelint index.js",
+    "prepublishOnly": "yarn test",
+    "test": "ava"
+  },
+  "devDependencies": {
```

Comment:
Should stylelint be a peerDependency?

Reply:
Yes my bad. Added to package.json of every custom plugin.